File: VB/WindowsApplication1/CustomEditor/AutoCompleteListEventArgs.vb
Imports System
Imports System.Collections.Generic

Namespace WindowsApplication1
	Public Class AutoCompleteListEventArgs
		Inherits EventArgs

		Public Sub New(ByVal autoSearchText As String, ByVal list As List(Of String))
			_AutoCompleteList = list
			_AutoSearchText = autoSearchText
		End Sub
		Private _AutoCompleteList As List(Of String)

		Private _AutoSearchText As String
		Public ReadOnly Property AutoSearchText() As String
			Get
				Return _AutoSearchText
			End Get
		End Property


		Public ReadOnly Property AutoCompleteList() As List(Of String)
			Get
				Return _AutoCompleteList
			End Get
		End Property
	End Class
End Namespace

File: CS/WindowsApplication1/CustomEditor/AutoCompleteListEventArgs.cs
using System;
using System.Collections.Generic;

namespace WindowsApplication1 {
    public class AutoCompleteListEventArgs : EventArgs {
        public AutoCompleteListEventArgs(string autoSearchText, List<string> list) {
            _AutoCompleteList = list;
            _AutoSearchText = autoSearchText;
        }
        private List<string> _AutoCompleteList;

        private string _AutoSearchText;
        public string AutoSearchText {
            get {
                return _AutoSearchText;
            }
        }


        public List<string> AutoCompleteList {
            get {
                return _AutoCompleteList;
            }
        }
    }
}
